build/project-template-gradle/src/main/java/com/tns/NativeScriptUncaughtExceptionHandler.java

```diff
@@ -11,25 +11,35 @@
 	private final UncaughtExceptionHandler defaultHandler;
 
 	private final Logger logger;
-
+	
+	private Runtime runtime;
+	
 	public NativeScriptUncaughtExceptionHandler(Logger logger, Context context)
 	{
 		this.logger = logger;
 		this.context = context;
 		defaultHandler = Thread.getDefaultUncaughtExceptionHandler();
 	}
+	
+	public void setRuntime(Runtime runtime)
+	{
+		this.runtime = runtime;
+	}
 
 	@Override
 	public void uncaughtException(Thread thread, Throwable ex)
 	{
 		String errorMessage = ErrorReport.getErrorMessage(ex);
 		
-		if (Platform.isInitialized())
+		if (Runtime.isInitialized())
 		{
 			try
 			{
 				ex.printStackTrace();
-				Platform.passUncaughtExceptionToJsNative(ex, errorMessage);
+				if (runtime != null)
```

Comment:
maybe minor but don't we need to fail here like sanity check. Also if runtime is available why go through the static Runtime.isInitialzied() if we require runtime instance set upfront anyway.